src/instructions/canis_lupus_meets_felis_silvestris.md

```diff
@@ -12,7 +12,7 @@
 
 Welcome to the *Canis lupus* meets *Felis silvestris* use case.
 
-In this example the GBIF occurrence data of *Canis lupus* and *Felis silvestris* are cut to the extent of Germany and linked to the land use classification of the Ecosystematlas.
+In this example the GBIF occurrence data of *Canis lupus* and *Felis silvestris* are cut to the extent of Germany and linked to the land use classification of the IÖR land use classification.
```

Comment:
Again, you could replace link with join already to introduce less terms. I won't comment on it again but you use "link" in multiple other places where it could be replaced by join.